packages/editor/src/components/dialogs/CreatePrefabPanelDialog.tsx

```diff
@@ -92,19 +100,31 @@ export default function CreatePrefabPanel({ entity }: { entity: Entity }) {
       const resource = resources.data[0]
       const tags = [...prefabTag.value]
       await Engine.instance.api.service(staticResourcePath).patch(resource.id, { tags: tags })
+
+      removeEntity(prefabEntity)
+      await EditorControlFunctions.removeObject([entity])
```

Comment:
calling removeObject twice. You don't need await, since the reactor handles the async logic